docs/UserGuide.adoc

```diff
@@ -284,7 +284,7 @@ Address book data are saved in the hard disk automatically after any command tha
 There is no need to save manually.
 
 === Upcoming `[coming in v2.0]`
-These features will be available in the next version of Mark.
+These features will be available in the next version of Mod Manager.
```

Comment:
For line 26 of this document, i think change Mark to Mod Manager. From line 33, 51 to 54, can help change ModManager to Mod Manager. Thanks.

Reply:
Resolved. Thanks for the feedback!